### PR TITLE
OKE Workload Identity Authentication

### PR DESCRIPTION
**Pull request by anders-swanson**

### CODE REVIEW

File: docs/src/main/asciidoc/core.adoc [hunk 1]
```diff
@@ -82,6 +82,16 @@ Set the config.type to `SESSION_TOKEN` as shown here.
...
 
+==== Workload Identity Configuration
+
+Set the config.type to `WORKLOAD_IDENTITY` as shown here.
+
+----
+spring.cloud.oci.config.type = WORKLOAD_IDENTITY
+----
+
+For further details, refer to https://blogs.oracle.com/cloud-infrastructure/post/oke-workload-identity-greater-control-access[OKE Workload Identity Authentication]
```
Comment (andytael) [inline docs/src/main/asciidoc/core.adoc:93]: Shouldn't we point to the docs instead? https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contenggrantingworkloadaccesstoresources.htm

**Review by andytael — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -15,7 +15,9 @@ For a deep dive into the project, refer to the Spring Cloud OCI documentation:
...
 |------------------------|---------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|
 | Spring Cloud OCI 1.0.0 | [Reference Docs](https://oracle.github.io/spring-cloud-oci/1.0.0/reference/html/index.html) | [API Docs](https://oracle.github.io/spring-cloud-oci/1.0.0/javadocs/index.html) |
-| Spring Cloud OCI 1.0.1 | [Reference Docs](https://oracle.github.io/spring-cloud-oci/1.0.0/reference/html/index.html) | [API Docs](https://oracle.github.io/spring-cloud-oci/1.0.0/javadocs/index.html) 
+| Spring Cloud OCI 1.0.1 | [Reference Docs](https://oracle.github.io/spring-cloud-oci/1.0.0/reference/html/index.html) | [API Docs](https://oracle.github.io/spring-cloud-oci/1.0.0/javadocs/index.html) |
+| Spring Cloud OCI 1.1.0 | [Reference Docs](https://oracle.github.io/spring-cloud-oci/1.0.0/reference/html/index.html) | [API Docs](https://oracle.github.io/spring-cloud-oci/1.0.0/javadocs/index.html) |
```
Comment (markxnelson) [inline README.md:19]: well we can add some more stuff, like adb, before we release 1.1.0, right, we revved to 1.1.0-SNAPSHOT, so i'd leave this line out until we do a release

**Review by markxnelson — CHANGES_REQUESTED** [review]
one minor comment

File: README.md (hunk 2 above)
Comment (anders-swanson) [reply README.md]: Updated!

**Review by markxnelson — APPROVED** [review]

**Pull request merged**